images/cluster-ingress-operator/Dockerfile

```diff
@@ -2,14 +2,15 @@ FROM centos:7
 
 RUN yum install -y golang make
 
+ENV CIO_PATH=src/github.com/openshift/cluster-ingress-operator
 ENV GOPATH /go
 RUN mkdir $GOPATH
 
-COPY . $GOPATH/src/github.com/openshift/cluster-ingress-operator
+COPY . $GOPATH/$CIO_PATH
 
-RUN cd $GOPATH/src/github.com/openshift/cluster-ingress-operator \
+RUN cd $GOPATH/$CIO_PATH \
     && make build \
-    && cp $GOPATH/src/github.com/openshift/cluster-ingress-operator/cluster-ingress-operator /usr/bin/
+    && cp $GOPATH/$CIO_PATH/_output/bin/cluster-ingress-operator /usr/bin/
```

Comment:
`_output/bin` isn't correct, seems like this Dockerfile should fail to build in its current state

Reply:
oh yeah i forgot to modify this after the rebase